### ISSUE TITLE
Should codemeta_validate be extended to validate a collection?

### ISSUE THREAD

**Issue by maelle** [ropensci/codemetar#156]
Currently `codemeta_validate` can only assess a JSON or a file corresponding to _one_ codemeta.json. Should it be extended to validate, say, a file like the rOpenSci corpus presented in https://ropensci.github.io/codemetar/articles/D-codemeta-parsing.html?

**Comment by cboettig** [comment]
Good point!  Yup, I should give that a go.  Note that validation is an intentionally more fuzzy concept in JSON-LD than it is with json schema or xml schema; because tools consuming JSON-LD will (or at least should) generally be more robust to variations in the input JSON-LD (e.g. it's okay to use additional terms that aren't part of the namespace; they don't break anything, they just get ignored by an any application that isn't interested in them).

**Comment by maelle** [comment]
When it's repaired, the codemeta.json of all rOpenSci packages might be a good use case for this. 😺
